src/entities/Place/types.ts

```diff
@@ -62,24 +60,24 @@ export type GetPlaceListQuery = {
   offset: string
   positions: string[]
   only_favorites: string
-  only_featured: string
   only_highlighted: string
   order_by: string
   order: string
   with_realms_detail: string
   search: string
+  categories: string[]
```

Comment:
why we accept `categories` as param when the attribute is call `category_id`?

Reply:
changed